### PR TITLE
adding basic support for yarn

### LINKED ISSUE #2: Remove `npm ci` from action?
Hey,

that action looks really nice, and I'd like to use it in our apps. However, we use yarn, so this fails on `npm ci`. 

It would be awesome if we could take the `npm ci` out of this action, and leave that to be done in the workflow? That way, it could be used in any environment. What do you think about this idea? 

PS: I've used this as a base for https://github.com/mydea/ember-cli-code-coverage-action/, thank you very much for the inspiration :)

### PR DESCRIPTION
**Pull request by mansona**
fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,14 @@ jobs:
...
+      with:
+        fetch-depth: 0
     - uses: simplabs/ember-asset-size-action@v1
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
 ```
+
+Note: as this action requires access to the "base" commit of a PR branch we need to fetch the whole repo by adding `fetch-depth: 0` to the `actions/checkout` configuration.
+
+If you would like to follow discussion on this issue you can follow: https://github.com/actions/checkout/issues/93
```
Comment (Turbo87) [inline README.md]: this seems unrelated to `yarn` support, doesn't it?

Comment (mansona) [reply README.md]: Yes but this suggested config broke 2 days ago because upstream changed something major (also why we're recommending targeting a tag instead of `@master` now).

Do you want me to split it into 2 PRs?

Comment (Turbo87) [reply README.md]: > Do you want me to split it into 2 PRs?

I think that would be appropriate 👍

Comment (mansona) [reply README.md]: @Turbo87 done: https://github.com/simplabs/ember-asset-size-action/pull/4
